streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamThread.java

```diff
@@ -475,6 +479,29 @@ private RuntimeException performOnAllTasks(final AbstractTaskAction action,
         return firstException;
     }
 
+    private RuntimeException performOnAllSuspendedTasks(final AbstractTaskAction action,
```

Comment:
These two functions are very similar: could we collapse them into one function `performOnTasks` and pass in a `List<AbstractTask>` as an additional parameters?